app/utility/arrays.ts

```diff
@@ -0,0 +1,26 @@
+namespace Utility
+{
+    export class Arrays
+    {
+        static FisherYatesShuffle<T>( array: Array<T> ): void
+        {
+            for ( var idx: number = array.length - 1; idx > 0; --idx )
```

Comment:
using `let` is preferred over using `var` just fyi. It has to do with scoping. If you use `var`, `idx` will still be in scope outside of the for loop. `let` is block scoped so it would not exists outside the for loop. just a js best practice. 

`for ( let idx: ...`

[see more here](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var#answers)